src/schema/v2/Alerts/index.ts

```diff
@@ -534,6 +539,10 @@ export const PartnerAlertsEdgeFields = {
         )
       }
 
+      // Make API call to fetch the first X collector profile records
+      const first = args.first ?? DEFAULT_COLLECTOR_PROFILES_BATCH_SIZE
+      const slicedUserIds = parent.user_ids.slice(0, first)
+
```

Comment:
Digging on the side here to see if I can pass `first:` into the gravity call

slicing what gets passed in the `user_ids` field in the API needs to be capped as well

Reply:
Yah was thinking there might be some weaving involved...